gateway/server.go

```diff
@@ -1582,6 +1582,13 @@ func Start() {
 		os.Exit(0)
 	}
 
+	// https://github.com/golang/go/issues/63413
```

Comment:
Pls add a bit more context to comment. That it required to enable RCA certificates.